lib/v2/screens/profile_screens/security/security_screen.dart

```diff
@@ -22,9 +24,37 @@ class SecurityScreen extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(title: Text('Security'.i18n)),
       body: BlocProvider(
-        create: (context) => SecurityBloc(),
+        create: (context) =>
+            SecurityBloc(settingsNotifier: SettingsNotifier.of(context), authNotifier: AuthNotifier.of(context))
+              ..add(const SetUpInitialValues()),
         child: MultiBlocListener(
           listeners: [
+            BlocListener<SecurityBloc, SecurityState>(
+              listenWhen: (previous, current) => previous.isSecurePin == false && current.isSecurePin == true,
+              listener: (context, state) {
+                showDialog<void>(
+                  context: context,
+                  barrierDismissible: false, // user must tap button
+                  builder: (_) => CustomDialog(
+                    icon: SvgPicture.asset('assets/images/security/success_outlined_icon.svg'),
+                    children: [
+                      Text(
+                        'Succesful'.i18n,
+                        style: Theme.of(context).textTheme.headline6.copyWith(color: AppColors.primary),
```

Comment:
these copy with on styles to add color is a problem. 
Lets bring it up to Jason in the next meeting.

Reply:
My mistake I just copy and paste this part. Its true we need tell this to Jason

Reply:
copyWiths removed